primitives/src/communities.rs

```diff
@@ -56,6 +56,19 @@ pub struct CommunityMetadata {
     pub url: Option<PalletString>,
 }
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub struct CidName {
+    pub cid: CommunityIdentifier,
+    pub name: PalletString,
+}
+
```

Comment:
Hmm, I don't quite agree that `CommunityName` would be a good name. I'd prefer `Community` as it is more than the name, but less than its metadata.

But `CidName` is awful. It hurts everytime I read it. XD

Reply:
ah, it's a double-identifier, I see. Then I think `CidName` is actually quite precise. No hurt in my eyes ;-)

Reply:
but `Community` is ok with me too. I don't want to make you suffer